client/state/sites/selectors.js

```diff
@@ -81,13 +82,17 @@ export const getSite = createSelector(
 			return null;
 		}
 
-		return {
+		const decoratedSite = {
 			...site,
-			...getComputedAttributes( site ),
-			...getJetpackComputedAttributes( state, siteId ),
 			hasConflict: isSiteConflicting( state, siteId ),
 			is_previewable: isSitePreviewable( state, siteId )
```

Comment:
Subtle difference here now in that `is_previewable` will be overriden by `getComputedAttributes`.

https://github.com/Automattic/wp-calypso/blob/57c1372/client/lib/site/computed-attributes.js#L44-L50

I think this is unintentional and undesirable.